src/client/package.json

```diff
@@ -122,6 +122,7 @@
     "eslint": "^8.28.0",
```

Comment:
maybe bump this to latest (8.36.0) .. and any other eslint-related packages

Reply:
would prefer to do this in the package upgrades branch i still have to complete after this gets merged.  react will need a different plugin version for 18